config/index.js

```diff
@@ -71,24 +71,4 @@ try {
   }
 }
 
-if (!process.env.USE_HUB_KUBECONFIG) {
-  if (
-    !nconf.get('options:hub:baseDomain') ||
-    !nconf.get('options:hub:user') ||
-    !nconf.get('options:hub:password')
-  ) {
-    throw new Error(`Missing environment variables.
-      The following are required to run this tests:
-      - OPTIONS_HUB_BASEDOMAIN or options.hub.baseDomain in options.yaml.
-      - OPTIONS_HUB_USER or options.hub.user in options.yaml.
-      - OPTIONS_HUB_PASSWORD or options.hub.password in options.yaml.\n`)
-  }
-} else {
-  if (!nconf.get('options:hub:kubeconfig')) {
-    throw new Error(`Missing environment variables.
-      The following are required to run this tests:
-      - OPTIONS_HUB_KUBECONFIG or options.hub.kubeconfig in options.yaml.`)
-  }
-}
-
```

Comment:
Removing `USE_HUB_KUBECONFIG` usage. It's much simpler to maintain the process of using the hub cluster credentials.